Input/Input.module.less

```diff
@@ -152,29 +164,34 @@
 			background-color: @agate-input-focused-background-color;
 			background-image: @agate-input-focused-background-image;
 			box-shadow: 0 0 0 @agate-input-decorator-focus-border-width @agate-input-decorator-focused-border-color;
+			border-color: @agate-input-active-border-color;
+			border-width: 0 0 @agate-input-decorator-border-bottom-width;
+			opacity: 1;
```

Comment:
Is this opacity necessary?

Reply:
It makes it so when the Input is disabled and focused it still has full opacity. The design does not seem to show a focused disabled input. If you wish I can remove it.

Reply:
I see. I think we could remove it. :)

Reply:
Done